internal/controller/typesensecluster_configmap.go

```diff
@@ -114,6 +121,9 @@ func (r *TypesenseClusterReconciler) updateConfigMap(ctx context.Context, ts *ts
 
 	availableNodes := len(nodes)
 	if availableNodes == 0 {
+		if ts.Spec.Pause {
+			return nil, 0, nil
```

Comment:
you could add smth here like r.logger.V(debugLevel).Info("skipping quorum configuration: cluster is paused")